src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -891,7 +891,6 @@ export default class Fms {
         }
 
         this._routeModel = nextRouteModel;
-        this.hasDepartureClearance = true;
```

Comment:
FYI just something I stumbled upon. Not used anywhere and departure clearance shouldn't be part of the FMS.

Reply:
good catch.... should be `Pilot.hasDepartureClearance`, not `Fms`.